docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/eclipse-code-ready-studio-guide/master.adoc
 
+:_content-type: PROCEDURE
```

Comment:
@stoobie I think the content type should be directly before the ID, not before the condition (ifdef). What do you think?

Reply:
@RichardHoch  I thought that in each condition the tag would be directly before ID. But maybe it would be better to put it twice  - in both conditions.